ui2/src/pages/login/QuickSetup.tsx

```diff
@@ -734,6 +758,8 @@ const CreateRoleContract = (props: {
             return acceptAllOffers(settlementOffers.contracts, SettlementOffer.Accept);
           case ServiceKind.LISTING:
             return acceptAllOffers(distributorOffers.contracts, DistributorOffer.Accept);
+          case ServiceKind.REGULATOR:
+            return acceptAllOffers(regulatorOffers.contracts, RegulatorOffer.Accept);
```

Comment:
is this not handled in the trigger?

Reply:
It does, but the quick setup page is currently working independently of the trigger afaict. The other offer accepts above are also handled by the trigger, if it is running